sidebars.js

```diff
@@ -1,13 +1,17 @@
+const sdkSidebar = require('./sdkSidebar.json')
+
 /**
  * Sidebar configuration.
  */
 
 module.exports = {
-  someSidebar: {
+  main: {
     'Getting Started': ['getting-started/create-comit-app'],
     'COMIT Protocol': ['comit-protocol/comit-protocol-stack'],
     'Core Concepts': ['core-concepts/atomic-swap-htlc', 'core-concepts/instant-swap', 'core-concepts/privacy-preserving-swap', 'core-concepts/negotiation'],
     'Tutorials': [],
-    'Tools': [],
+    'SDK': sdkSidebar,
```

Comment:
```suggestion
    'COMIT Javascript SDK': sdkSidebar,
```

Or just `Javascript SDK` - I feel just SDK is a bit naked. 
The idea of just dropping the `Tools` one hierarchy above is fine for me.